vuepress/docs/next/docs/getting-started/ent-profile.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebarDepth: 1
+---
+
+
+# Profile Management
+
+The [Entando CLI](entando-cli.md) can define a collection of configuration variables to full describe an Entando instance. One or more profiles can be created to store configuration settings for the instance. The `ent profile` command is available to manage and switch between the configurations of different Entando instances. Refer to `ent profile first-use-readme` for additional details. A profile with this information is automatically generated for a quickstart application.
+
+At minimum, a configuration must consist of the key-value pairs specifying the application name and namespace. To run the application, the Kubernetes connection must also be provided. Cloud Kubernetes tools typically create a Kube context, which can be linked to an Entando profile. An overview of the current connection and profile information is provided via `ent status`.
+
+The following commands perform common operations associated with profile management.
+
+## New Profile
+
+Create and switch to a new profile:
+```
+ent pro new [profileName] [EntandoAppName] [namespace]
+```
+This sets the minimal profile data and outputs next steps.
+
+## Link Profile
+
+Link the current profile to a Kubernetes context (alias of `ent attach-kubectx`):
+```
+ent pro link [contextName]
+```
+This provides ent with instructions to connect to the Kubernetes containing the Entando Application.
+
+## Global Profile
+
+Activate the profile that ent should use across shells:
+```
+ent pro use [profileName]
+```
+This attempts to fully restore a profile by informing the login and related Entando applications.
```

Comment:
This sentence doesn't feel quite right. Maybe, based on the next command, "Activate the profile ent should use by default."